css/styles.css

```diff
@@ -1,14 +1,62 @@
-*,#map{
+#map{
   margin:0;
   padding:0;
-  height:100%;
+  height:100vh;
 }
 
 .scrollFix {
     line-height: 0.1;
     overflow: hidden;
     white-space: nowrap;
     float:left;
-    padding-bottom: 0px;
-    margin-bottom: 0px;
+    padding-bottom: 0;
+    margin-bottom: 0;
 }
+
+.gm-style-iw {
+  max-height: 200px;
+}
+
+#iw-container {
+  margin-bottom: 10px;
+}
+#iw-container .iw-title {
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-size: 22px;
+  font-weight: 400;
+  padding: 10px;
+  background-color: #48b5e9;
+  color: white;
+  margin: 0;
+  border-radius: 2px 2px 0 0;
+}
+#iw-container .iw-content {
+  font-size: 13px;
+  line-height: 18px;
+  font-weight: 400;
+  margin-right: 1px;
+  padding: 15px 5px 20px 15px;
+  max-height: 140px;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+.iw-content img {
+  float: right;
+  margin: 0 5px 5px 10px;
+}
+.iw-subTitle {
```

Comment:
sometimes you name classes using camel case sometimes its dash separated - choose one ;)